src/API/Base.php

```diff
@@ -455,32 +455,23 @@ public static function update_merchant_feed( $merchant_id, $feed_id, $args ) {
 	 * Get a specific merchant's feed using the given arguments.
 	 *
 	 * @param string $merchant_id The merchant ID the feed belongs to.
-	 * @param string $feed_id     The ID of the feed to be updated.
+	 * @param string $feed_id     The ID of the feed.
 	 *
 	 * @return mixed
 	 */
 	public static function get_merchant_feed( $merchant_id, $feed_id ) {
-		$args = array( 'feed_profile' => $feed_id );
-
 		return self::make_request(
-			add_query_arg( $args, 'catalogs/datasource/feed_report/' . $merchant_id . '/' ),
-			'GET'
+			'catalogs/datasource/feed_report/' . $merchant_id . '/',
+			'GET',
+			array(
+				'feed_profile' => $feed_id
```

Comment:
Minor: This needs a comma at the end.

```suggestion
				'feed_profile' => $feed_id,
```